contrib/completion/zsh/_docker

```diff
@@ -226,8 +226,9 @@ __docker_subcommand() {
         "($help)*--cap-drop=-[Drop Linux capabilities]:capability: "
         "($help)--cidfile=-[Write the container ID to the file]:CID file:_files"
         "($help)*--device=-[Add a host device to the container]:device:_files"
-        "($help)*--dns=-[Set custom dns servers]:dns server: "
-        "($help)*--dns-search=-[Set custom DNS search domains]:dns domains: "
+        "($help)*--dns=-[Set custom DNS servers]:DNS server: "
+        "($help)*--dns-opt=-[Set custom DNS options]:DNS option: " \
```

Comment:
oops!  Fixed and pushed

On Fri, Sep 11, 2015 at 1:43 AM, Steve Durrheimer <notifications@github.com>
wrote:

> In contrib/completion/zsh/_docker
> <https://github.com/docker/docker/pull/16031#discussion_r39251701>:
>
> > @@ -226,8 +226,9 @@ __docker_subcommand() {
> >          "($help)*--cap-drop=-[Drop Linux capabilities]:capability: "
> >          "($help)--cidfile=-[Write the container ID to the file]:CID file:_files"
> >          "($help)*--device=-[Add a host device to the container]:device:_files"
> > -        "($help)*--dns=-[Set custom dns servers]:dns server: "
> > -        "($help)*--dns-search=-[Set custom DNS search domains]:dns domains: "
> > +        "($help)*--dns=-[Set custom DNS servers]:DNS server: "
> > +        "($help)*--dns-opt=-[Set custom DNS options]:DNS option: " \
>
> Typo, need to remove the \ (this section of code is a array).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/16031/files#r39251701>.
>